thavalon-server/src/connections/account_handlers.rs

```diff
@@ -0,0 +1,309 @@
+//! Rest handlers for account-based calls
+use super::validation::{self, JWTResponse, RefreshTokenInfo, TokenStore, ValidationError};
+use crate::database::{self, account_errors::AccountError, DatabaseAccount};
+use chrono::{DateTime, NaiveDateTime, Utc};
+use serde::{Deserialize, Serialize};
+use serde_json;
+use warp::{
+    http::{response::Builder, StatusCode},
+    reject::{self, Reject},
+    reply, Rejection, Reply,
+};
+
+/// Canonical representation of a Thavalon user.
+/// This struct is safe to send from the database, as it does not contain a password hash.
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct ThavalonUser {
+    pub email: String,
+    pub password: String,
+    #[serde(default)]
+    pub display_name: String,
+    pub profile_picture: Option<Vec<u8>>,
```

Comment:
Let's choose either Option or serde(default)

Reply:
This comment is actually fair.... which makes me sad.

Reply:
After discussion, these structs are going to be split for better granularity which will solve the problem, but it's a significant change. Created issue #24 to track this.